scripts/ci/is-scanner-v2-available.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
This must be run with `set -e` otherwise failures in this file will not cause the exit code to be >0.

Reply:
Good catch, I missed this. just fixed it and I'll check the build log more carefully this time.